bookings/templates/bookings/booking_confirmed.html

```diff
@@ -2,48 +2,49 @@
 
 {% block content %}
 
-<!-- Confirmation of booking -->
-<div class="container-fluid px-0 customer-bookings">
-    <div class="row booking-content booking-confirmation">
-        <div class="col-12 col-md-10 col-lg-8 col-xl-6 mb-5 mx-auto bg-color-white booking-card txt-dark lead">
-            <div class="row">
-                <div class="col-12">
-                    <h3 class="mt-2"><strong>Thank You!</strong></h3>
-                    <hr class="m-0">
-                    <p class="mt-3">We have a spare table and your booking has been made.</p>
-                    <p>Your booking details are as follows:</p>
-                </div>
-            </div>
-            <div class="row">
-                <div class="col-12 col-sm-6 mt-2 mb-4">
-                    <div class="booking-image-cont"></div>
-                </div>
-                <!-- Details of booking -->
-                <div class="col-12 col-sm-6 mt-2 booking-info">
-                    <p>The Pizza Oven</p>
-                    <p>Table for {{ booking.party_size }} people</p>
-                    <p>{{ booking.date }} at {{ booking.time|time:"H:i" }}</p>
+    <!-- Confirmation of booking -->
```

Comment:
While I have a general dislike of in line commenting, I think in HTML files it can really aid the maintainer in finding things more easily